src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -49,6 +52,19 @@ private struct Element
             public const string GridHeightAnimator = "gridHeightAnimator";
         }
 
+        /// <summary>
+        /// TextBlock, which is shown during graph calculation.
+        /// TODO: this should be replaced with some animated text. 
+        /// http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-9584
+        /// </summary>
+        private readonly TextBlock busyTextBlock = new TextBlock
+        {
+            Text = Configurations.BusyString,
+            FontFamily = new FontFamily("Consolas"),
+            Opacity = 0.5,
+            Margin = new System.Windows.Thickness(12, 6, 0, 10)
+        };
+
```

Comment:
Hi @aosyatnik, I have had a chat with @kronz, it seems like that's not a good enough justification for us to complicate the code further. So when @Racel has a chance to talk that through with Zach, we'll probably just close that task out :)